packages/compass-indexes/src/components/index-header-column/index-header-column.jsx

```diff
@@ -53,11 +52,15 @@ class IndexHeaderColumn extends PureComponent {
       <th
         data-test-id={this.props.dataTestId}
         className={this._renderClassName()}
-        onClick={this.handleIndexSort.bind(this)}>
+        onClick={this.handleIndexSort.bind(this)}
+      >
         {this.props.name}
         <i
-          className={classnames(styles['index-header-column-sort'], `fa fa-fw ${this.props.sortOrder}`)}
-          onClick={this.handleIndexSort.bind(this)}/>
```

Comment:
Removed this `onClick` as it was on a non-interactive element and the parent already has an `onClick`.